.github/docker/Dockerfile.packaging-centreon-collect-alma8

```diff
@@ -0,0 +1,97 @@
+ARG REGISTRY_URL
+
+FROM ${REGISTRY_URL}/almalinux:8 AS centreon_collect_packaging
+
+RUN bash -e <<EOF
+
+# Base dnf configuration
+sed -i 's/best=True/best=False/g' /etc/dnf/dnf.conf
+echo 'install_weak_deps=False' >> /etc/dnf/dnf.conf
+
+dnf install -y dnf-plugins-core
+dnf config-manager --set-enabled powertools
+dnf install -y epel-release
+
+echo '[goreleaser]
+name=GoReleaser
+baseurl=https://repo.goreleaser.com/yum/
+enabled=1
+gpgcheck=0' | tee /etc/yum.repos.d/goreleaser.repo
+
+dnf install --best -y \
+    git \
+    lua-devel \
+    make \
+    selinux-policy-devel \
+    yum-utils \
+    perl-ExtUtils-Embed \
+    perl-HTTP-Daemon-SSL \
+    perl-interpreter \
+    perl-JSON \
+    perl-Thread-Queue \
+    procps-ng \
+    nfpm-2.41.1 \
+    sudo \
+    wget \
+    zstd
+
+dnf update -y libarchive
```

Comment:
does it require an update command specifically instead of install (as I think install attempts to update a package if it already exists)

Reply:
let s leave it as it is for now, to avoid any unforeseen consequences